go/oasis-test-runner/scenario/e2e/runtime/runtime_dynamic.go

```diff
@@ -63,7 +78,12 @@ func (sc *runtimeDynamicImpl) Fixture() (*oasis.NetworkFixture, error) {
 	}
 	// Test storage genesis state for compute runtimes. Also test with a non-zero round.
 	f.Runtimes[1].GenesisRound = 42
-	f.Runtimes[1].GenesisState = "tests/fixture-data/runtime-dynamic/runtime-genesis-state.json"
+	f.Runtimes[1].GenesisState = storage.WriteLog{
+		{
+			Key:   mustUnbase64("Z2VuZXNpcyBzdGF0ZQ=="),
+			Value: mustUnbase64("aGVsbG8gd29ybGQ="),
```

Comment:
```suggestion
			Key:   []byte("genesis state"),
			Value: []byte("hello world"),
```
:grin:

Reply:
Also, should make these strings constants as they are tested to be equal below on lines 263+.